src/Frontend/Page/Admin.hs

```diff
@@ -222,7 +222,7 @@ instance FormPage PageAdminSettingsDurations where
     formAction _ = relPath $ U.Admin U.AdminDuration
 
     -- FIXME: Do we redirect to the same page???
-    redirectOf _ = relPath $ U.Admin U.AdminDuration
+    redirectOf _ _ = relPath $ U.Admin U.AdminDuration
```

Comment:
When matching `()`, I prefer the more specific pattern `()` over `_`.  Makes it more readable and gives you more guidance when the type changes and you want to react to it.

This has some implications re. strictness, but I don't think that matters here.

Just a comment, also I'm curious what you all think. let's leave it as is for now.